playbooks/common/openshift-cluster/upgrades/v3_9/label_nodes.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+- hosts: oo_first_master
+  roles:
+    - lib_openshift
+  post_tasks:
+    - when: osm_default_node_selector is not defined
+      block:
+        - name: Retrieve non-infra, non-master nodes
+          oc_obj:
+            state: list
+            kind: Node
+            selector: region!=infra,node-role.kubernetes.io/master!=true
```

Comment:
We allow folks to specify their infra selector which defaults to region=infra see https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_hosted/defaults/main.yml#L5

Can we set a var here with the same defaulting and then s/=/!=/ ?

Reply:
Yeah no problem, would there ever be a case where they specify their `openshift_hosted_infra_selector` as `!=`, in which case I assume `!!=` would break stuff?